db/seeds.rb

```diff
@@ -5,3 +5,93 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+FactoryBot.define do
+  factory :user do
+    username {Faker::Internet.unique.user_name(5..15)}
+    password_digest {BCrypt::Password.create(123456)}
+    created_at {Faker::Time.between(5.years.ago, Date.today)}
+    updated_at {created_at}
+  end
+end
+
+FactoryBot.define do
+  factory :post do
+    user_id {1}
+    text {
+      if rand < 0.2
+        Faker::Movies::HitchhikersGuideToTheGalaxy.quote
+      elsif rand < 0.4
+        Faker::Movies::Lebowski.quote
+      elsif rand < 0.6
+        Faker::TvShows::BojackHorseman.quote
+      elsif rand < 0.8
+        Faker::TvShows::DrWho.quote
+      end
+    }
+    created_at {DateTime.now}
+    updated_at {created_at}
+  end
+end
+
+FactoryBot.define do
+  factory :like do
+    user_id {1}
+    post_id {1}
+    created_at {DateTime.now}
+    updated_at {created_at}
+  end
+end
+
+FactoryBot.define do
+  factory :subscription do
+    subscriber_id {1}
+    subscribed_to_id {1}
+    created_at {DateTime.now}
+    updated_at {created_at}
+  end
+end
+
+
+users = []
+1000.times do
+  users << FactoryBot.build(:user)
+end
+User.import users
+
+posts = []
+User.all.each do |user|
+  rand(1...100).times do
+    posts << FactoryBot.build(:post,
+                              user_id: user.id,
+                              created_at: Faker::Time.between(user.created_at, Date.today))
+  end
+end
+Post.import posts
+
+posts_count = Post.all.count
+
+likes = []
+User.all.each do |user|
+  Faker::Number.unique.clear
+  rand(1...500).times do
+    likes << FactoryBot.build(:like,
+                              user_id: user.id,
+                              post_id: Faker::Number.unique.between(1, posts_count))
```

Comment:
Этот код слишком оптимистично полагает, что все id от 1 и до количества постов существуют. Порой это может быть не так.
У тебя есть массив posts в котором находятся все посты которые были созданы. Используй его
```suggestion
                              post: posts.sample)
```